util/fipstools/delocate/delocate.peg

```diff
@@ -100,6 +100,7 @@ ARMGOTLow12 <- ":got_lo12:" SymbolName
 ARMPostincrement <- '!'
 BaseIndexScale <- '(' RegisterOrConstant? WS? (',' WS? RegisterOrConstant WS? (',' [0-9]+)? )? ')'
 Operator <- [+\-]
-Offset <- '+'? '-'? (("0b" [01]+) / ("0x" [[0-9A-F]]+) / [0-9]+)
+OperatorOffset <- '+' / '-' / '*'
```

Comment:
Nit: If this is the operator acting on the offset, should it be called `OffsetOperator`?

Reply:
Fixed. Thanks.